app/assets/javascripts/date_chooser.coffee

```diff
@@ -40,16 +38,9 @@ class DateChooser
     selector.html(selector.text())
     @setDate selector
     @enableLink otherSelector
-    @dayInput.focus()
```

Comment:
Maybe we could tab focus to the date input after Today/Tomorrow clicked?

We could replace the `@values` variable with a `@dateInput` variable, and use that in the rest of the code, e.g.
```ruby
@dateInput = chooser.find(".date-chooser-values").find(".date")
...
@dateInput.focus()
...
@dateInput.val(date)
```